src/components/shared/Accordion.js

```diff
@@ -6,19 +7,32 @@ const Svg = require("./Svg");
 
 require("./Accordion.css");
 
+type AccordionItem = {
+  buttons?: Array<Object>,
+  component: () => any,
+  header: string,
+  opened: boolean,
+  onToggle?: () => any,
+  shouldOpen?: () => any
+};
+
+type Props = {
+  items: Array<Object>
+};
+
 const Accordion = React.createClass({
   propTypes: {
-    items: PropTypes.array
+    items: PropTypes.array.isRequired
   },
 
   displayName: "Accordion",
 
-  getInitialState: function() {
+  getInitialState() {
     return { opened: this.props.items.map(item => item.opened),
       created: [] };
   },
 
-  componentWillReceiveProps: function(nextProps) {
+  componentWillReceiveProps(nextProps: Props) {
```

Comment:
I did some investigation. It seems files in the `shared` folder that have `componentWillReceiveProps `will have this error if we don't have a type for `nextProps`.

Those files are `Accordion.js` and `ManagedTree.js`.